src/metadatacache/MetadataCache.ts

```diff
@@ -93,49 +86,22 @@ export class MetadataCache {
    * @param  {SearchQuery} query Query to filter the DDOs.
    * @return {Promise<QueryResult>}
    */
-  public async queryMetadata(query: SearchQuery): Promise<QueryResult> {
-    const result: QueryResult = await this.fetch
-      .post(`${this.url}${apiPath}/query`, JSON.stringify(query))
+  public async queryMetadata(query: SearchQuery): Promise<any> {
+    const result = await this.fetch
+      .post(`${this.url}/api/v1/aquarius/assets/query`, JSON.stringify(query))
```

Comment:
should work with the `apiPath` const defined at top of file, otherwise defining that as const becomes redundant if not used whenever we have an API path

Reply:
yeah, somebody thought than instead of /api/v1/aquarius/assets/ddo/query to use /api/v1/aquarius/assets/query :(